packages/plugin-ext/src/api/rpc-protocol.ts

```diff
@@ -329,7 +329,8 @@ class MessageFactory {
         if (typeof res === 'undefined') {
             return `{${prefix}"type":${MessageType.Reply},"id":"${req}"}`;
         }
-        return `{${prefix}"type":${MessageType.Reply},"id":"${req}","res":${JSON.stringify(res, ObjectsTransferrer.replacer)}}`;
+        const cache: any = [];
```

Comment:
Why do you need to pass in the `cache` if it has the default params? I have proposed the API change, which you have made, to avoid any client code changes. Did I overlook something?

Reply:
Need to store cache out of the function, otherwise the array would be overwritten with an empty array each time the `replacer` function is called.

Reply:
I see, thanks! 👍